data_access_log/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class DataAccessLog(models.Model):
+    """Model that contains all the logged information."""
+    model = models.CharField('Accessed Model', max_length=100, null=False, blank=False)
+    viewset = models.CharField('Accessed ViewSet', max_length=100, null=False, blank=False)
+    timestamp = models.DateTimeField('Acess timestamp', auto_now_add=True, editable=False, null=False, blank=False)
+    user = models.CharField('Accessing user', max_length=255, null=False, blank=False)
+    url = models.CharField('URL', max_length=255, null=False, blank=False)
+    response = models.TextField('Response', null=False, blank=False)
+
+    def save(self, *args, **kwargs):
+        # Only creation allowed
```

Comment:
stops saving silently? should produce some error

Reply:
It is done in the official documentation: https://docs.djangoproject.com/en/2.1/topics/db/models/#overriding-predefined-model-methods

Reply:
Yep, strange. Ok!